doc/api/net.md

```diff
@@ -515,56 +544,74 @@ changes:
     description: The `hints` option is supported now.
 -->
 
-Opens the connection for a given socket.
-
-For TCP sockets, `options` argument should be an object which specifies:
-
-  - `port`: Port the client should connect to (Required).
-
-  - `host`: Host the client should connect to. Defaults to `'localhost'`.
+* `options` {Object}
+* `connectListener` {Function} Common parameters of [`socket.connect()`][]
+  methods. Will be added as a listener for the [`'connect'`][] event once.
+* Returns: {net.Socket} The socket itself.
 
-  - `localAddress`: Local interface to bind to for network connections.
+Initiate the connection on a given socket.
 
-  - `localPort`: Local port to bind to for network connections.
+Note: this method is similar to a [connect(2)][] call, and before that,
```

Comment:
> so binding to an IP addr is pretty much "binding to an interface", you figure out what interface you want, figure out its address, then bind to the address

Yeah but saying "Local interface to bind to" for a `localAddress` property is a bit confusing..one might wonder can Node guess the IP for us in a clever way when we simply pass in something like "eth0"? Or do we have to figure it out by ourselves first then give the IP to Node ? (Until you get an error saying it only takes an IP, it doesn't take interface names..but then the doc can be clear about this in the first place :P)

I don't feel strongly about the man pages, it's just a note..I'll remove that if this proves unhelpful before(in fs.md)